tasks/src/question-answering/data.ts

```diff
@@ -43,25 +43,25 @@ const taskData: TaskData = {
 	libraries: TASKS_MODEL_LIBRARIES["question-answering"],
 	metrics:   [
 		{
-			description: "Exact Match is a metric that is based on the strict character match of the predicted answer and the ground truth. For correct answers, EM is 1. Even if one character is different, EM will be 0.",
+			description: "The Exact Match metric is based on the strict character match of the predicted answer and the right answer. For correct predicted answers, the Exact Match will be 1. Even if only one character is different, the Exact Match will be 0.",
 			id:          "exact-match",
 		},
 		{
-			description: " F1-Score is a useful metric if we value both false positives and false negatives equally. F1-Score is calculated over each word in the predicted sequence against the answer in the ground truth.",
+			description: " The F1-Score metric is useful if we value both false positives and false negatives equally. The F1-Score is calculated on each word in the predicted sequence against the correct answer.",
 			id:          "f1",
 		},
 	],
 	models: [
 		{
-			description: "A strong baseline model for most question answering domains.",
+			description: "A robust baseline model for most question-answering domains.",
```

Comment:
I'd rather have:
A robust baseline model for most question answering domains.

Reply:
Agree, will apply :)